### PR TITLE
Video 42 list orders

### PR DESCRIPTION
**Pull request by basir**

### CODE REVIEW

File: frontend/src/screens/OrderListScreen.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              <th>PAID</th>
+              <th>DELIVERED</th>
+              <th>ACTIONS</th>
+            </tr>
+          </thead>
+          <tbody>
+            {orders.map((order) => (
+              <tr key={order._id}>
+                <td>{order._id}</td>
+                <td>{order.user.name}</td>
```
Comment (AhmadHussain1) [inline frontend/src/screens/OrderListScreen.js:41]: screenshot png

Comment (AhmadHussain1) [reply frontend/src/screens/OrderListScreen.js:41]: i am stuck here . it can't show user name . how to solve this.

Comment (NguyenNgocPhuongUyen) [reply frontend/src/screens/OrderListScreen.js:41]: you can replace {order.user.name} by {order.shippingAddress.fullname}

Comment (SachinthaLakshan) [reply frontend/src/screens/OrderListScreen.js:41]: check your DB and delete orders came from deleted users

File: frontend/src/screens/OrderListScreen.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    onClick={() => {
+                      props.history.push(`/order/${order._id}`);
+                    }}
+                  >
+                    Details
+                  </button>
+                  <button
+                    type="button"
+                    className="small"
+                    onclick={() => deleteHandler(order)}
```
Comment (sentinel-sam) [inline frontend/src/screens/OrderListScreen.js:63]: `onClick={() => deleteHandler(order)}`

**Pull request merged**